include/zephyr/tracing/tracing.h

```diff
@@ -1765,12 +1765,14 @@
 /**
  * @brief Trace System heap realloc enter
  * @param heap
+ * @param ptr
```

Comment:
I would add a description pof the parameter here, even if it seems very obvious, see below...